edx/frontend-app-learner-portal
# added client side variables for auth

**Description**: This PR adds client side variables for authentication. Gatsby requires that any environment variables required to be pushed to the client side must have a GATSBY_ prefix. The following plugin 
https://github.com/chadly/gatsby-plugin-env-variables
is used to ensure that the frontend-auth related variables already defined in the edx-internal repo donot have to be replicated with the above prefix.

**JIRA**: https://openedx.atlassian.net/browse/ENT-1925
**Test instructions:**
1: Browse to environment/app/1
2: If you not already logged into lms, you should be redirected to lms login page
3: Log into lms and get redirected back to the learner portal
4: Repeat after logging out of lms
5: Repeat when already logged into lms
